common/live-streaming.md

```diff
@@ -69,7 +69,8 @@ By default, the live stream is composed in landscape mode for desktop viewers (w
 To integrate 100ms Interactive Live Streaming in your app, follow these steps:
 
 1. [Enable live streaming destination](#enable-destination)
-1. [Integrate the 100ms SDK in your app](#sdk-integration)
+2. [Integrate the 100ms SDK in your app](#sdk-integration)
+3. [Integrate playback for the audience](#hls-playback-for-the-audience)
```

Comment:
link is wrong

Reply:
fixed. [Preview](https://100ms-docs-mv7w5w3qf-100mslive.vercel.app/docs/javascript/v2/foundation/live-streaming#video-on-demand-vod-use-cases)